drivers/unix/file_access_unix_pipe.cpp

```diff
@@ -65,7 +65,7 @@ Error FileAccessUnixPipe::open_internal(const String &p_path, int p_mode_flags)
 	struct stat st = {};
 	int err = stat(path.utf8().get_data(), &st);
 	if (err) {
-		if (mkfifo(path.utf8().get_data(), 0666) != 0) {
+		if (mkfifo(path.utf8().get_data(), 0600) != 0) {
```

Comment:
Why? Pipes are only useful for IPC, I would expect it to be accessible by every process.

Reply:
I would only trust the processes running as my user, or users I explicitly give permission to. If I was playing a game that uses pipes for IPC, on a shared home computer, and my smart little sister figured out that she could prank me by sshing in and writing garbage to /tmp/game-pipe to crash the game then I would complain to the game developer.

The main purpose for pipes at the moment seems to be to support `OS.execute_with_pipe()`, which only needs 0600 permissions because the subprocess runs as the same user. If devs create pipes directly and want to loosen the access then they could use `FileAccess.set_unix_permissions()`.